scale/source/models.py

```diff
@@ -8,6 +8,7 @@
 from django.utils.timezone import now
 
 import storage.geospatial_utils as geo_utils
+from job.models import Job
```

Comment:
I wasn't sure if it's okay for the source app to depend on job or if it should be inlined?